public/app/features/browse-dashboards/components/RecentlyDeletedEmptyState.tsx

```diff
@@ -13,7 +13,10 @@ export const RecentlyDeletedEmptyState = ({ searchState }: RecentlyDeletedEmptyS
       message={
         userIsSearching
           ? t('recently-deleted.page.no-search-result', 'No results found for your query')
-          : t('recently-deleted.page.no-deleted-dashboards', "You haven't deleted any dashboards recently.")
+          : t(
+              'recently-deleted.page.no-deleted-dashboards',
+              "You haven't deleted any dashboards recently. When you delete a dashboard, it will appear here for 30 days before being permanently deleted. Your organization administrator can restore recently-deleted dashboards."
```

Comment:
When seeing the text altogether, it seems too much for me:

<img width="672" alt="Captura de pantalla 2024-07-17 a las 10 30 45" src="https://github.com/user-attachments/assets/fe346da4-3107-428b-b725-65eba8667b5a">

What if we split it and have a main phrase, "You haven't deleted any dashboards recently", and then a secondary explanation as it is here:

<img width="418" alt="Captura de pantalla 2024-07-17 a las 10 30 54" src="https://github.com/user-attachments/assets/32d067b2-a14a-4e60-9284-4fa6c4c7052a">

WDYT?

Reply:
Good catch, looks way better now!
![Screenshot 2024-07-17 at 14 29 41](https://github.com/user-attachments/assets/a005ba7b-ab11-4fbc-9695-5fa83f31b80a)